app-misc/sl/sl-5.02.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Amusing graphic for mistyping habits"
+DESCRIPTION="SL (Steam Locomotive) runs across your terminal when you type \
+\"sl\" as you meant to type \"ls\". It's just a joke command, and not useful \
+at all."
+HOMEPAGE="https://github.com/mtoyoda/sl"
+COPYRIGHT="1993,1998,2014 Toyoda Masashi"
+LICENSE="Toyoda"
+REVISION="1"
+SOURCE_URI="https://github.com/mtoyoda/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="1e5996757f879c81f202a18ad8e982195cf51c41727d3fea4af01fdcbbb5563a"
+SOURCE_DIR="sl-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	sl = $portVersion
+	cmd:sl = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libncurses
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libncurses
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	cp sl $binDir
+
+	mkdir -p $dataDir/sl
+	cp LICENSE $dataDir/sl
```

Comment:
No need for this LICENSE="Toyoda" takes care of putting the license in $dataDir/licenses :)
```diff
-
-	mkdir -p $dataDir/sl
-	cp LICENSE $dataDir/sl
```